events/events.go

```diff
@@ -0,0 +1,251 @@
+
+	tokenResponse, err := CallHttpMethod("POST", loginUrl+"/oauth/token", headers, strings.NewReader(data), true)
+	defer tokenResponse.Body.Close()
+	if err != nil {
+		fmt.Printf("Error while getting access-token (CF-Login-CURL call)")
+		return "", err
+	} else {
+		bodyBytes, err2 := ioutil.ReadAll(tokenResponse.Body)
+		if err2 != nil {
+			fmt.Printf("Error while decoding curl response")
+			return "", err2
+		}
+		var s map[string]string
+		err = json.Unmarshal(bodyBytes, &s)
+		accessToken := s["access_token"]
+		return accessToken, nil
+	}
+}
+
+func (c *EventCommand) ListEvents(cliConnection plugin.CliConnection, noInstanceNames bool, action string) {
+	Initialize()
+	fmt.Println("Getting the list of instance events in the org", AddColor(helper.GetOrgName(helper.ReadConfigJsonFile()), constants.Cyan), "/ space", AddColor(helper.GetSpaceName(helper.ReadConfigJsonFile()), constants.Cyan), "...")
+	var cmd string
+	var actionType string
+	var userSpaceGuid = helper.GetSpaceGUID(helper.ReadConfigJsonFile())
+
+	if helper.GetAccessToken(helper.ReadConfigJsonFile()) == "" {
+		errors.NoAccessTokenError("Access Token")
+	}
+
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetBorder(false)
+	table.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.SetCenterSeparator(" ")
+	table.SetColumnSeparator(" ")
+	table.SetRowSeparator(" ")
+	table.SetHeaderLine(false)
+	table.SetAutoFormatHeaders(false)
+
+	table.SetHeader([]string{AddColor("instance_name", constants.White), AddColor("instance_guid", constants.White), AddColor("event_type", constants.White), AddColor("user", constants.White), AddColor("created_at", constants.White)})
+
+	var columns int = 5
+	var field = make([]string, columns)
+
+	if action == "create" {
+		actionType = "audit.service_instance.create"
+		cmd = "/v2/events?q=type:" + actionType + "%3Bspace_guid:" + userSpaceGuid
+	} else if action == "update" {
+		actionType = "audit.service_instance.update"
+		cmd = "/v2/events?q=type:" + actionType + "%3Bspace_guid:" + userSpaceGuid
+	} else if action == "delete" {
+		actionType = "audit.service_instance.delete"
+		cmd = "/v2/events?q=type:" + actionType + "%3Bspace_guid:" + userSpaceGuid
+	} else {
+		cmd = "/v2/events?q=type+IN+audit.service_instance.delete,audit.service_instance.create,audit.service_instance.update%3Bspace_guid:" + userSpaceGuid
+	}
+	var AuthorizationEndpoint string = helper.GetLoginEndpoint(helper.ReadConfigJsonFile())
+	var apiEndpoint string = helper.GetApiEndpoint(helper.ReadConfigJsonFile())
+	var refreshToken string = helper.GetRefreshToken(helper.ReadConfigJsonFile())
+
+	accessToken, _ := GetAccessToken(AuthorizationEndpoint, refreshToken, "refresh_token")
+	curlResponse, err := ExecuteCurl(apiEndpoint, accessToken, cmd)
+
+	if err != nil {
+		fmt.Println(AddColor("FAILED", constants.Red))
+		fmt.Printf("Errors in getting Orgs: ", err)
```

Comment:
As asked in previous review as well [here](https://github.com/cloudfoundry-incubator/service-fabrik-cli-plugin/pull/2#discussion_r198436162), why is this error in getting orgs? Shouldn't it be "Error in getting service instance events"?